src/components/BulkEnrollmentPage/index.jsx

```diff
@@ -1,15 +1,43 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import Hero from '../Hero';
+import { Container } from '@edx/paragon';
 
+import { Switch, Route } from 'react-router-dom';
+import Hero from '../Hero';
+import { MultipleSubscriptionsPage, SubscriptionData } from '../subscriptions';
 import CourseSearch from './CourseSearch';
 
 function BulkEnrollmentPage({ enterpriseId }) {
   return (
     <div className="container-fluid bulk-enrollment">
       <Hero title="Catalog Management" />
-      <CourseSearch enterpriseId={enterpriseId} />
+      <SubscriptionData enterpriseId={enterpriseId}>
+        <main role="main" className="manage-subscription">
+          <Switch>
+            <Route
+              path="/:enterpriseSlug/admin/catalog_management"
```

Comment:
nit: Should the route use a hyphen vs. underscore? "catalog-management" vs. "catalog_management" as its a bit odd to mix and match hyphens/underscores in the URL when the enterprise slug and other routes use hyphens. User-facing URLs typically only use hyphens.

Reply:
Yeah, hyphens sound good to me. :)